modules/ROOT/pages/release-notes.adoc

```diff
@@ -5,3 +5,24 @@
 ====
 The 2023.1 is in development.
 ====
+
+== Feature deprecations and removals
+
+=== Removals
+
+==== Multi-tenancy removal
+
+Deprecated xref:2022.2@ROOT:multi-tenancy-and-tenant-configuration.adoc[multi-tenancy] has been removed.
```

Comment:
Here I would add an indication that it concerns only Subscription edition installations.

Reply:
done